README.md

```diff
@@ -31,10 +31,13 @@ where Rails.config.zipkin_tracer or config is a hash that can contain the follow
  * `:whitelist_plugin` - plugin function which recieves Rack env and will force sampling if it returns true
  * `:zookeeper` - plugin function which uses zookeeper and kafka instead of scribe as the transport
  * `:logger` - A logger class following the standard's library Logger interface (Log4r, Rails.logger, etc).
+ * `json_api_host` - hostname with protocol of a zipkin api instance (e.g. `https://zipkin.example.com`) to use JSON over HTTP instead of Scribe or Kafka
+ * `traces_buffer` (default: 10) - the number of annotations stored until automatic flush
```

Comment:
Actually we run our clients with a buffer of `1000`. I stuck to the `scribe_max_buffer` default here, how does `100` sound then?
Note that eventually the scribe tracer is going to be phased out (isn't it?), that is the reason I added this new configuration key. With a broader name it could be used in the kafka tracer too, if it ever implements some kind of buffering.